packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -34,7 +34,16 @@ import {
 } from 'utils/document/nonContractUtils';
 import { withErrorBoundary, WithErrorBoundaryProps } from 'utils/hoc/withErrorBoundary';
 import { Filter, FilterGroup, FilterChangeArgs } from '../FilterPanel/types';
-import { MetadataData, EnrichedHtml, Contract, Item, Field } from 'components/CIDocument/types';
+import {
+  Metadata,
+  MetadataData,
+  EnrichedHtml,
+  Contract,
+  Item,
+  Field,
+  Attributes,
+  Relations
+} from '../../types';
```

Comment:
do we need to do `../../` here? i thought that `components/CIDocument/types` was more readable